README.md

```diff
@@ -25,27 +25,3 @@ The ctClient can then be used to access the Cherrytwist server api using provide
 The capabilities provided by the Cherrytwist server api is continually expanding, so not all capabillities of the server are guaranteed to be exposed by this package. For full details of the graphql api exposed by the Cherrytwist server please browse the schema for the graphql api via our [Dev environment](http://dev.cherrytwist.org/graphql).
 
 The set of wrapper calls provided is based on needs to date; feel free to augment with new ones as the needs arise.
-
-## Authentication: Acquire Token with Resource Owner Password Credentials flow (ROPC)
```

Comment:
Where does the capability to connect command line now reside with this removed? It does not seem to be in the tokenator library which may be where it logically belongs. 

In particular it might still be useful to have documentation for how one could use the tokenator package to be able to do command line obtaining of a token - and we may want to have this capability wrapped in our client package, even if the actual authentication code resides in tokenator - thoughts?

Reply:
Could also put the ability to connect in an authenticated fashion into our populator package

Reply:
There is enough description in the tokenator page already, I don't believe it makes sense to define which env variables need to be acquired and mandate some specific config - that's for us to decide.